.github/workflows/images.yaml

```diff
@@ -0,0 +1,129 @@
+name: Build
+
+on:
+  push:
+    tags:
+      - '*'
+      - '!JENKINS*'
```

Comment:
Given that we are currently struggling with this syntax in the PyPi workflow, I checked the documentation and it seems to be one of the ways to allow/disallow actions based on the tag name.
Another option would be:
```
tags-ignore:
 - 'JENKINS*'
```

as documented in: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore

Just an FYI @khurtado